ArjunaJTA/jta/tests/classes/com/hp/mwtests/ts/jta/commitmarkable/TestCommitMarkableResourceBase.java

```diff
@@ -61,7 +61,7 @@ public final void setup() throws Exception {
 
 		resetPropertiesFile = System
 				.getProperty("com.arjuna.ats.arjuna.common.propertiesFile");
-		if (resetPropertiesFile == null) {
+		if (resetPropertiesFile == null || !resetPropertiesFile.equals("commitmarkableresourcejbossts-properties.xml")) {
```

Comment:
The name and location of the properties file is no longer configurable. If that is the intention then just hard code it directly and don't do any getProperty calls or tests:

`System.setProperty("com.arjuna.ats.arjuna.common.propertiesFile", 					"commitmarkableresourcejbossts-properties.xml"); `